modular_skyrat/modules/stories_system/code/story_datums/somewhat_impactful.dm

```diff
@@ -82,3 +82,20 @@
 	poor_sod = null
 	return ..()
 
+/*
+	NRI R&R
+		Plot Summary:
+			A small group of NRI marines are on shore leave, so they got drunk and decided to land on this station, ideally to get more drunk.
```

Comment:
I kinda wish they weren't military (I mean come on literally every NRI-station interaction is military-related good god this makes me very-very sad :( ) but this is very subjective and oh you do you.

Reply:
I can add a NRI tourist if we've got clothes for them

Reply:
Like, we have an entire sprite pack of unused NRI diplomats' drip and an unfinished NRI diplomatic mission ERT. The latter could be removed and reused as a story instead. I think it'd fit way better than another NRI military group of pepol.

Reply:
sure, PR them to the branch and I'll accept it

Reply:
> sure, PR them to the branch and I'll accept it

Oh nice brb.